### PR TITLE
[price-pusher] support for aptos

### PR DESCRIPTION
**Pull request by 0xfirefist**

### CODE REVIEW

File: package-lock.json
Comment (guibescos) [inline package-lock.json:1]: Feels like too many changes here for just adding 1 package. What version of npm are you using?

File: price_pusher/README.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ npm run start -- injective --grpc-endpoint https://grpc-endpoint.com \
     [--pushing-frequency 10] \
     [--polling-frequency 5] \
 
+# For Aptos
```
Comment (guibescos) [inline price_pusher/README.md:74]: Nitpick :
Update `endpoint`,  `price-service-endpoint`, `pyth-contract-address`  to match `config.aptos.testnet.sample.json`

File: price_pusher/src/aptos/command.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    const mnemonic = fs.readFileSync(mnemonicFile, "utf-8").trim();
+
+    const priceItems = priceConfigs.map(({ id, alias }) => ({ id, alias }));
+
+    const pythListener = new PythPriceListener(
+      priceServiceConnection,
+      priceItems
+    );
+
+    // create aptos listerer and pusher
```
Comment (guibescos) [inline price_pusher/src/aptos/command.ts:72]: `listerer` -> `listener`

**Review by guibescos — COMMENTED** [review]
I really like how you reused parts of the other pushers. That's awesome.
Main comment is figuring out this package-lock.json thing. Seems like a lot of integrate hashes disappeared 
image

File: package-lock.json
Comment (0xfirefist) [reply package-lock.json:1]: 8.19.3

Comment (0xfirefist) [reply package-lock.json:1]: I updated the package-lock.json with the latest commit. Can you please take a look?

File: price_pusher/src/aptos/command.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    } as Options,
+    ...options.priceConfigFile,
+    ...options.priceServiceEndpoint,
+    ...options.mnemonicFile,
+    ...options.pythContractAddress,
+    ...options.pollingFrequency,
+    ...options.pushingFrequency,
+  },
+  handler: function (argv: any) {
+    // FIXME: type checks for this
```
Comment (ali-bahjati) [inline price_pusher/src/aptos/command.ts:37]: I saw we have it for all the CLIs. Maybe.. just ignore it and remove this fixme?

**Review by ali-bahjati — APPROVED** [review]
Nice and clean ser 🔥

File: package-lock.json
Comment (guibescos) [reply package-lock.json:1]: All good now thanks

**Review by guibescos — APPROVED** [review]

**Review by guibescos — APPROVED** [review]

**Review by guibescos — APPROVED** [review]

**Pull request merged**